tests/test_api.py

```diff
@@ -111,12 +111,16 @@ def test_train_on_cached_data(self, bliss_client, pretrained_weights_filename):
     def test_predict_sdss_default_rcf(self, bliss_client, weight_save_path):
         bliss_client.predict_sdss(
             weight_save_path=weight_save_path,
-            predict={"device": "cpu"},
+            predict={"device": "cpu", "trainer": {"accelerator": "cpu"}},
```

Comment:
Shouldn't we use a GPU if the user has one?

Reply:
Note the conftest.py overrides the device in `testing_config.yaml` if pytest is run with the `--gpu` flag:
https://github.com/prob-ml/bliss/blob/8b6cbb7f6e9d1ebf7e1094dc0def5d60de905171/tests/conftest.py#L36-L37

Reply:
Ah okay I see. Yep we'll do something similar to test on GPU